core/tests/protractor/editorAndPlayer.js

```diff
@@ -577,12 +577,70 @@ describe('Full exploration editor', function() {
       'TextInput', forms.toRichText('Happy!'), null, false, 'Equals',
       'happy');
     explorationEditorMainTab.expectInteractionToMatch('TextInput');
+    explorationEditorPage.saveChanges();
     explorationEditorMainTab.deleteInteraction();
     explorationEditorMainTab.setInteraction('EndExploration');
     explorationEditorMainTab.expectInteractionToMatch('EndExploration');
     users.logout();
   });
 
+  it('should play the recommended exploration successfully', function() {
```

Comment:
@seanlip Has this feature been tested? This test spec adds exploration ID as recommendation into the "End Exploration" interaction and then play-test to make it sure the recommended exploration display.

Reply:
I don't know if it's been tested, but I'm happy to add a test if not :)